libraries/GCS_MAVLink/GCS_MAVLink.h

```diff
@@ -45,10 +45,14 @@ extern mavlink_system_t mavlink_system;
 /// @param chan		Channel to send to
 static inline bool valid_channel(mavlink_channel_t chan)
 {
+#if defined __clang__
```

Comment:
we already suppress warnings for unknown pragmas... so this is not really needed. Or are you still getting the warnings?